### DISCUSSION TITLE
My GitHub Repository not showing any Languages.

### DISCUSSION THREAD

**Discussion by kartiks123** [github-linguist/linguist#6720] [Classification]
I have created a private repository and in it, no "Language" section is being shown. I learnt something about creating a '.gitattributes' file, but what should I include in it generally, in order to detect all the languages being used.
Any lead would be highly appreciated.

Sorry for the less information provided, but due to some reasons, I am not willing to share the file content of this repository.

**Comment by lildude** [comment]
It's impossible to tell you what to put in your `.gitattributes` file without seeing the repo as the overrides require knowing file names or paths or extensions. See the [Overrides](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md) docs for more details.

That said, I doubt you'll need to implement overrides.

> I have created a private repository and in it

This suggests to me this may be a very new repo with very few pushes, maybe even only one. It is known for the background job that does the analysis to sometimes not run on very new repos so pushing a few more commits will probably fix things.
 
If that doesn't work, it's probably because your repo only contains content linguist ignores by default or is in a directory linguist ignores by default. 
Read [How Linguist Works](https://github.com/github-linguist/linguist/blob/master/docs/how-linguist-works.md) and follow the links to files and view the content you'll be able to determine if any of the that applies to you and then you'll need to either make changes or implement overrides.

Short of that, we can't help without seeing the repo. Feel free to invite me as a collaborator (I work for GitHub) and I can take a look if you like.

**Reply by kartiks123** [reply]
Sure sir, I think my research was less than I expected. I'll look into the docs, probably it will be resolved.

**Comment by Lacters** [comment]
BUENOS DÍAS

SOLO SE CONTINUA CON EL PROTOCOLO PARA EL BIEN DE LA FUNCIONALIDAD DEL
SOFTWARE  LE REITERO MIS RESPETOS.

GRACIAS.
